databases/tiledb/Portfile

```diff
@@ -74,11 +74,6 @@ if {[string match *gcc* ${configure.compiler}] && ${configure.build_arch} in [li
                     patch-libatomic-32-bit.diff
 }
 
-# Patch TileDB to find lz4 manually, because the lz4 package does not provide CMake configs.
-# Updating the lz4 port is blocked on lz4 releasing a new version, with some official
-# linkage-agnostic targets.
-patchfiles-append   lz4_fix.diff
```

Comment:
Aside from the issue which I pointed out in the ticket, there are extra changes needed here, since now patches do not apply. We also need to add a dependency on `libmagic` from looks of things, and `libpng` (?).

I have already fixed patches, I will make a commit and you can pick that.

Reply:
Libmagic is being obtained through the dependency on the `file` port, and libpng is optionally needed for some tests and there is a dependency to it in the test variant (see line 138 in the new portfile).

Reply:
Ok, then only fixing the second patch is left.